packages/library-legacy/src/connection.ts

```diff
@@ -6291,10 +6290,7 @@
     args: IWSRequestParams,
   ): ClientSubscriptionId {
     const clientSubscriptionId = this._nextClientSubscriptionId++;
-    const hash = fastStableStringify(
-      [subscriptionConfig.method, args],
-      true /* isArrayProp */,
```

Comment:
Pretty sure this never did anything. TypeScript passed because this function used to be `any`.